### ISSUE TITLE
Chips not accessible with a screen reader

### ISSUE THREAD

**Issue by marcysutton** [angular/material#8897]
**Actual Behavior**:
While chips work great with the keyboard alone, they are inaccessible with screen readers for a few reasons:
1. The chips control is hidden with `aria-hidden="true"`, so it essentially doesn't exist for a screen reader user. How will they interact with chips without some other affordance?
2. There is no way to focus on existing chips with the arrow keys when focused in an input field like you can with the keyboard. 
3. Pressing delete sends focus to the last chip, but nothing is announced; probably because the entire control has `aria-hidden="true"`. 
4. The demo input fields are not labeled.

**What is the expected behavior?**
Chips should be usable with a screen reader running. Users should be able to determine what a chip is, how many there are, and how to delete them without seeing the screen.

This component needs an accessible UX review to decide the appropriate roles and interactions for screen reader users. Off the top of my head I'm not sure what the right roles would be–ARIA menus and tabs aren't really appropriate, even if they might help with the arrow keys in a screen reader. You're focused on an input field the whole time, so it's not immediately obvious what the right pattern would be. Keep in mind the responsive nature of this framework means it should also work with a screen reader on a touch device, but I recognize even getting it to work on desktop will be a challenge.

**Details** (or steps to reproduce the issue): *
Visit the Chips demo on https://material.angularjs.org in Safari or Chrome with Voiceover running. On Windows, the issue is reproducible with NVDA and JAWS. 

Screenshot: Arrow keys in a screen reader do not select chips. Focus is in the input field, but virtual cursor is on h2 heading after pressing the left arrow key

<img width="1429" alt="Arrow keys in Voiceover do not select chips" src="https://cloud.githubusercontent.com/assets/1045233/16435849/f4916f74-3d4d-11e6-8d87-06e75bc43d4f.png">

**Angular Versions**: *
-  Angular Version:
-  Angular Material Version:
  All

**Additional Information**:
-  `Browser Type: *`
-  `Browser Version: *`
-  `OS: *`
-  `Stack Traces:`
  All

---

Shortcut to create a [new CodePen Demo](http://codepen.io/team/AngularMaterial/pen/bEGJdd).
Note: `*` indicates required information. Without this information, your issue may be auto-closed.

> Do not modify the titles or questions. Simply add your responses to the ends of the questions.
>   Add more lines if needed.
